src/ng/sce.js

```diff
@@ -341,6 +341,9 @@ function $SceDelegateProvider() {
      * Takes the result of a {@link ng.$sceDelegate#trustAs `$sceDelegate.trustAs`} call and
      * returns the originally supplied value if the queried context type is a supertype of the
      * created type.  If this condition isn't satisfied, throws an exception.
+     * 
+     * **WARNING:** Disabling auto-escaping is extremely dangerous, it usually creates a Cross
+     * Site Scripting (XSS) vulnerability in your application.
```

Comment:
You can wrap the content in a `<div class="alert alert-warning/danger">...</div>`.
Example: https://docs.angularjs.org/api/ng/directive/ngInit